src/PKSim.Core/Chart/SimulationTimeProfileChart.cs

```diff
@@ -0,0 +1,9 @@
+﻿using OSPSuite.Core.Chart;
+
+namespace PKSim.Core.Chart
+{
+   public class SimulationTimeProfileChart : AnalysisChart
```

Comment:
This was somewhat of a duplcate class. One is abstract in core now. This is the implementation